src/IconButton/_IconButton.scss

```diff
@@ -33,6 +33,7 @@ $btn-icon-accent-color: white !default;
 
     &.focus,
     &:focus {
+        outline: 0;
```

Comment:
Please change to 2px transparent; this will show up in Windows High Contrast mode, when box-shadow treatments are automatically removed.